src/Semver/SemVersion.cs

```diff
@@ -129,21 +110,13 @@ public static SemVersion Parse(string version, bool strict = false)
             if (!match.Success)
                 throw new ArgumentException("Invalid version.", "version");
 
-#if NETSTANDARD
```

Comment:
@niemyjski you are probably right. Although I cannot think of language dependent differences in integer parsing - there may be some.